modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1554,6 +1556,21 @@ export class Wallet implements IWallet {
     });
   }
 
+  /**
+   *  Sign a message
+   * @param params
```

Comment:
Can we add more context for "Sign a message"? It may be too vague for external users.
```suggestion
   * @param {WalletSignMessageOptions} params
   * @returns {Promise<SignedMessage>}
```

Reply:
I also have the same concern; can we elaborate on what a basic `sign a message` means? also the `@param` type as well

Reply:
This has been addressed. 
![image](https://user-images.githubusercontent.com/99677485/187753701-7023e79c-4c9a-4b05-8dec-38e5e97b1fbe.png)